doc/radosgw/notifications.rst

```diff
@@ -94,7 +96,10 @@ Request parameters:
 
 - Kafka endpoint 
 
- - URI: ``kafka://<fqdn>[:<port]``
+ - URI: ``kafka://[<user>:<password>@]<fqdn>[:<port]``
+ - if ``ca-location`` is provided, secure connection will be used for connection with the broker
+ - user/password may only be provided over HTTPS. Topic creation request will be rejected if not
```

Comment:
specifically here:  user/password may only be provided via secure transport (ok, good), but then "user/password may only be provided together with ca-location"--I'm not understanding the connection between the user/password and ca-location, shouldn't they be orthogonal?

Reply:
> specifically here: user/password may only be provided via secure transport (ok, good), but then "user/password may only be provided together with ca-location"--I'm not understanding the connection between the user/password and ca-location, shouldn't they be orthogonal?

the meaning is that user/password can *configured* during topic creation over secure HTTP, and *used*, during notification sending over secure kafka connection
however, I agree that ca-location is not the correct method for indicating that kafka connection will be secure.
will add a separate flag for that (e.g. use-ssl), and have ca-location optional, as the default CA location may also be used in some cases

Reply:
fixed

Reply:
done